frontend/src/stimulus/controllers/pattern-input.controller.ts

```diff
@@ -189,10 +202,7 @@ export default class PatternInputController extends Controller {
   private setRange():void {
     const selection = document.getSelection();
     if (selection?.rangeCount) {
-      const range = selection.getRangeAt(0);
-      if (range.startContainer.parentNode === this.contentTarget) {
-        this.currentRange = range;
-      }
+      this.currentRange = selection.getRangeAt(0);
```

Comment:
The change here _could_ mean that we don't even need a `currentRange` local variable anymore. It was meant to keep track of a valid range (as in, a range that is not inside a token).

Worth checking if I'm not forgetting something else, though.

Reply:
Hmm, I thought we need this variable to maintain the range, when I get out of the input with `ArrowDown` to select a completion element? Is my assumption wrong?